es-app/src/MetaData.cpp

```diff
@@ -8,6 +8,7 @@ namespace fs = boost::filesystem;
 MetaDataDecl gameDecls[] = {
 	// key,         type,                   default,            statistic,  name in GuiMetaDataEd,  prompt in GuiMetaDataEd
 	{"name",        MD_STRING,              "",                 false,      "name",                 "enter game name"},
+	{"sortname",    MD_STRING,              "",                 false,      "sortname",             "enter game sort name"},
```

Comment:
I just want to note, adding new items in the metadata XML is a big change; there are many 3rd party scripts and tools  (scrapers, etc,)  that edit and work with it. I do not have a good overview of what are the current 'users' of this structure that might need to be informed of this change.